### PR TITLE
Docs/alpha onboarding

### PR DESCRIPTION
**Pull request by oblm**
#### Description
This PR is a duplicate of https://github.com/spatialos/UnrealGDK/pull/442. It is necessary because we're not setup to accept external PRs into master, as you can't run external PR code through the build gate:
`ci/spatialos/UnrealGDK/windows/pre-merge`
#### Tests
None
#### Documentation
This is documentation.
#### Primary reviewers
@danielimprobable

### CODE REVIEW

File: SpatialGDK/Documentation/content/handover-between-server-workers.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
-
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/handover-between-server-workers.md]: lets ignore this file for this PR too as I think mike has looked into this one

Comment (danielimprobable) [reply SpatialGDK/Documentation/content/handover-between-server-workers.md]: @m-samiec

File: SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md]: @joshuahuburn has a PR in flight for this, lets leave this one for his PR

File: SpatialGDK/Documentation/content/singleton-actors.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/singleton-actors.md]: @m-samiec has a PR in flight for this doc lets remove it from this PR

Comment (m-samiec) [inline SpatialGDK/Documentation/content/singleton-actors.md]: Please remove this file also, I have these changes in a separate PR.

File: SpatialGDK/Documentation/content/toolbar.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/toolbar.md]: revert the changes in this file, have a PR for this already merged

File: SpatialGDK/Documentation/content/troubleshooting.md [hunk 5]
```diff
@@ -30,7 +28,7 @@ Validate that the SpatialOS entity that represents your Actor appears in the Ins
 
 **Q:** I’m getting the following compilation error when building the GDK: `Error C2248: FRepLayout::Cmds': cannot access private member declared in class 'FRepLayout`.
 
-**A:** You're building against an unsupported version of Unreal Engine. Make sure you're targeting the fork of Unreal Engine that the GDK requires. See the [setup guide]({{urlRoot}}/setup-and-installing#building) for more details.
+**A:** You're building against an unsupported version of Unreal Engine. Make sure you're targeting the fork of Unreal Engine that the GDK requires. See the [setup guide]({{urlRoot}}/content/get=started/2-build-unreal) for more details.
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/troubleshooting.md]: This link seems wrong

File: SpatialGDK/Documentation/known-issues.md [hunk 6]
```diff
@@ -1,10 +1,8 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
-
 # GDK for Unreal known issues
 
 Known issue = any major user-facing bug or lack of user-facing feature that:
 1. diverges from vanilla Unreal design or implementation (e.g. ordering of reliable RPCs), **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+2. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/known-issues.md]: should remain 1.

File: SpatialGDK/Documentation/toc.md [hunk 7]
```diff
@@ -1,12 +1,16 @@
 - <h3>[- SpatialOS GDK for Unreal]({{urlRoot}}/index)</h3>
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/toc.md]: @ElleEss has prepared an update for this one already, lets not merge this file

File: SpatialGDK/Documentation/content/troubleshooting.md [hunk 8]
```diff
@@ -8,8 +6,8 @@
 
 **A:** There could be a few different reasons for this. The list below provides some of the most common ones, ordered by likelihood:
 1. It's easy to forget to generate the schema for your replicated Actor. Make sure you run the Schema Generator before launching your project..
-1. As per Unreal Engine’s [replication documentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors), your Actor needs to be created on the server-worker before it can replicate to the client-workers.
-1. Ensure that your call to `SpawnActor` is happening on your server-worker.
+2. As per Unreal Engine’s [replication documentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors), your Actor needs to be created on the server-worker before it can replicate to the client-workers.
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/troubleshooting.md]: should be using 1.s

File: SpatialGDK/Documentation/content/get-started/3-gdk-and-starter-project.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+4. On the toolbar, click **Play** to run the game.
+5. When you're done, click **Stop** to stop the client [worker](https://docs.improbable.io/reference/latest/shared/glossary), and click the **Stop** button in the SpatialOS section of the toolbar to stop your local SpatialOS deployment.
+
+### Deploy the Starter Project in the cloud
+
+To run a cloud deployment, you need to prepare your server-worker and client-worker [assemblies](https://docs.improbable.io/reference/latest/shared/glossary), and upload them to the cloud.
+
+> Building the assemblies can take a while - we recommend installing <a href="https://www.incredibuild.com/" data-track-link="Incredibuild|product=Docs|platform=Win|label=Win" target="_blank">IncrediBuild</a> to speed up build times.
+
+1. Change the name of the project
```
Comment (danielimprobable) [inline SpatialGDK/Documentation/content/get-started/3-gdk-and-starter-project.md]: these should be using 1.s
